STYLE_API_TEMPLATE.md

```diff
@@ -104,9 +132,9 @@ Upon success, the response returns "acknowledged": true
 }
 ```
 
-### Request body
+### Response body
 
-For PUT and POST APIs: Introduce what the request fields are allowed to provide in the body of the request.
+Provide meanings for the fields returned in the response.
 
```

Comment:
remove all remaining lines - it is a duplicate section for the Request Fields descriptions.

Reply:
Agree with Alice, but we decided to keep the Response section and name it "Response Fields"

Reply:
Also, it could be relocated right after the Request fields section (with table of field descriptions).